components/automate-ui/src/app/pages/+compliance/shared/reporting/report-query.service.spec.ts

```diff
@@ -48,4 +48,12 @@ describe('ReportQueryService', () => {
       expect(service.getFilterTitle(profileType, id)).toEqual(profileTitle);
     });
   });
+
+  describe('formatReturnParams', () => {
+    it('returns empty object when there are no filters and date is today ', () => {
+      service.formatReturnParams();
+      // this is just testing an empty report Query
+      expect(service.formatReturnParams()).toEqual({});
+    });
+  });
```

Comment:
I could use some help figuring out how to Mock a Report Query response so that I can make this test better.  Right now its just testing that an empty call comes back empty, which doesn't really do much.